docs/pypi-replication.md

```diff
@@ -24,81 +25,126 @@ P_30                  	all	0.3102
-  E.g., `msmarco-doc` should be change to  `ms-marco-doc`. Exactly same results are expected to obtain.
-  
-- WARNING: In `pyserini==0.10.0.0`, hits are hard coded to 1000. see [here](https://github.com/castorini/pyserini/blob/pyserini-0.10.0.0/pyserini/search/__main__.py#L110).
-  Even though for MS MARCO doc we only evaluate top 100.
-  In `pyserini==0.10.0.1`, the hits are has been parameterized. see [here](https://github.com/castorini/pyserini/blob/pyserini-0.10.0.1/pyserini/search/__main__.py#L112).
-  
-CAVEAT: current HEAD of master is getting different results.
+$ python ms_marco_doc_eval.py --judgments qrels.msmarco-doc.dev.txt --run run.msmarco-doc.doc.txt
+#####################
+MRR @100: 0.2770296928568709
+QueriesRanked: 5193
+#####################
 ```
-MS MARCO passage ranking task, BM25 baseline:
-map                   	all	0.1958
-recall_1000           	all	0.8573
 
-MS MARCO passage ranking task, BM25 baseline with docTTTTTquery expansions:
-map                   	all	0.2893
-recall_1000           	all	0.9506
+MS MARCO document ranking task, BM25 baseline with [docTTTTTquery expansions](http://doc2query.ai/) (per-document):
+
+```bash
+$ python -m pyserini.search --topics msmarco_doc_dev --index msmarco-doc-expanded-per-doc --output run.msmarco-doc.doc-expanded.txt --bm25 --hits 100 --msmarco
+```
+
+Evaluation command:
+
+```bash
+$ python ms_marco_doc_eval.py --judgments qrels.msmarco-doc.dev.txt --run run.msmarco-doc.doc-expanded.txt
+#####################
+MRR @100: 0.32651902964919355
+QueriesRanked: 5193
+#####################
+```
+
+MS MARCO document ranking task, BM25 baseline, but with documents segmented into passages and selecting the best-scoring passage per document:
+
+```bash
+$ python -m pyserini.search --topics msmarco_doc_dev --index msmarco-doc-per-passage --output run.msmarco-doc.passage.txt --bm25 --hits 1000 --max-passage --max-passage-hits 100 --msmarco
+```
+
+Evaluation command:
+
+```bash
+$ python ms_marco_doc_eval.py --judgments qrels.msmarco-doc.dev.txt --run run.msmarco-doc.passage.txt
+#####################
+MRR @100: 0.2751202109946906
+QueriesRanked: 5193
+#####################
+```
+
+MS MARCO document ranking task, BM25 baseline with [docTTTTTquery expansions](http://doc2query.ai/) (per-passage):
 
-MS MARCO Document Ranking
-map                   	all	0.2775
-recall_1000           	all	0.9357
+```bash
+$ python -m pyserini.search --topics msmarco_doc_dev --index msmarco-doc-expanded-per-passage --output run.msmarco-doc.passage-expanded.txt --bm25 --hits 1000 --max-passage --max-passage-hits 100 --msmarco
 ```
 
+Evaluation command:
+
+```bash
+$ python ms_marco_doc_eval.py --judgments qrels.msmarco-doc.dev.txt --run run.msmarco-doc.passage-expanded.txt
+#####################
+MRR @100: 0.32081861579183807
+QueriesRanked: 5193
+#####################
+```
+
+## Notes
+
+Prior to v0.10.1.0, the above commands get different results:
```

Comment:
we can add a link here point to the v0.10.0.1 replication doc to show the results before?

Reply:
Yes, good call. Will do.